### ISSUE TITLE
Loss Nan when training the mode

### ISSUE THREAD

**Issue by mlkk518** [swimmiing/SCR-Torch#1]
Thanks for sharing your wonderful work! The training goes into failure case after several training iterations, and 'Nan' is printed. Though I have set the learning rate to 1e-8, and clip the gradient, the issue still exists. Could you please help me go against the problem. Thank you very much!   Best regards.
